### ISSUE TITLE
sudo npm install homebridge-gpio-wpi2 with -g not possible?

### ISSUE THREAD

**Issue by StefanMeGit** [rsg98/homebridge-gpio-wpi2#57]
I installed homebridge with -g on my raspberry pi (rasbian with deskopt and magicMirror) and cannot install homebride-gpio-wpi2 with sudo npm -g install homebridge-gpio-wpi2

is it not possible to isntall wpi2 for all users with-g?

```
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp"
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/.node-gyp/9.11.2"
```
​

**Comment by creativecat** [comment]
Similar issue here
Even when calling *sudo npm install -g --unsafe-perm homebridge-gpio-wpi2* I got an error:
```
make: *** [Release/obj.target/epoll/src/epoll.o] Error 1
make: Leaving directory '/usr/local/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:188:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:254:12)
gyp ERR! System Linux 4.14.79-v7+
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll
gyp ERR! node -v v11.7.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! epoll@0.1.22 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the epoll@0.1.22 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-01-24T20_18_02_894Z-debug.log
```

**Comment by jo-foe** [comment]
with me it also looks like this, is it possible that epoll is only executable supports Node.js versions 4, 6, 8 and 10 and not for 11?

**epoll.target.mk:101: recipe for target 'Release/obj.target/epoll/src/epoll.o' failed**
make: *** [Release/obj.target/epoll/src/epoll.o] Error 1
make: Leaving directory '/usr/local/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:197:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:254:12)
gyp ERR! System Linux 4.4.50-v7+
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/homebridge-gpio-wpi2/node_modules/epoll
gyp ERR! **node -v v11.9.0**
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! epoll@0.1.22 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! **Failed at the epoll@0.1.22 install script.**
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-02-07T08_19_12_329Z-debug.log

**Comment by rsg98** [comment]
Fixed in ceb1993
